airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -1417,6 +1417,14 @@
   icon: youtube.svg
   sourceType: api
   releaseStage: beta
+- sourceDefinitionId: 78752073-6d96-447d-8a93-2b6953f3c787
```

Comment:
Could we start for new sources to use human readable ids as outlined in https://github.com/airbytehq/airbyte/issues/18447. @sherifnada @evantahler While we might not want to switch all existing because of the migration pain, is there any reason why new connectors couldn't follow a new format, e.g. having an ID of `com.airbyte.youtube-analytics-business` as an ID?

Reply:
this new source will exist for only a few days.... can we pass it as is because it will be removed right after the Google approves new scope for Youtube analytics for business.

Reply:
yes sorry I wasn't aware of the demo nature of this when I wrote this. Please feel free to ignore for this connector.

Reply:
@timroes I believe a large amount of the java codebase would need to change because it expects the UUID format for all definitions. See https://github.com/airbytehq/airbyte/blob/master/airbyte-api/src/main/openapi/config.yaml#L2956